backend/census_historical_migration/sac_general_lib/general_information.py

```diff
@@ -310,14 +311,15 @@ def xform_auditee_fiscal_period_end(general_information):
 
 def xform_auditee_fiscal_period_start(general_information):
     """Constructs the fiscal period start from the fiscal period end"""
-    # Transformation to be documented.
-    fiscal_start_date = xform_census_date_to_datetime(
+    fiscal_end_date = xform_census_date_to_datetime(
```

Comment:
I suggest adding a comment to this method to explain what the code was doing before this change and to highlight when and why this fix was implemented. Specifically, this fix was applied after the migration, so it's unlikely that the code will be run again for migration purposes. However, it was decided that everything related to the migration, including this transformation code change, should be treated as public records. Therefore, it will be helpful to indicate to reader that all migrations were run with the old version of the code.